### DISCUSSION TITLE
High-value datasets and DCAT

### DISCUSSION THREAD

**Discussion by AlainVagner** [opendatateam/udata#2956] [Ideas]
Hello, 
we have seen in the [latest roadmap](https://www.data.gouv.fr/fr/pages/roadmap/), that you intend to have high-value datasets easily available in udata.
I was wondering if this could also encompass specific metadata for the dcat-ap endpoint, which would help to identify the high-value datasets.
There is a specific extension of [DCAT-AP for high-value datasets ](https://semiceu.github.io/DCAT-AP/releases/2.2.0-hvd/)
Could this be an interesting feature? 
Best regards,

**Comment by maudetes** [comment]
Hello,

Thank you for this discussion!
Indeed, we plan to support exposing specific HVD metadata as described in DCAT-AP HVD :)
It may be exposed on the existing DCAT catalog endpoint, or may be added on a specific HVD DCAT one (with datasets filtered on a HVD property/tag, etc.).

You may know however that some metadata can't be properly modeled in udata currently but we're working on it.
The most challenging one is API / DataService modelisation. You can look up the [discussions here](https://github.com/etalab/data.gouv.fr/issues/1294) (in French sorry).
We have also worked on the [ContactPoint](https://github.com/opendatateam/udata/pull/2914) as well, that is now supported (in backend only).

We'd be curious about your thoughts on this topic!

Best regards

**Comment by AlainVagner** [comment]
Hello @maudetes, 
thanks for your detailed answer. This looks really promising! 
For the endpoints, as long as we are able to get the information, it should be fine. In 10.8 of the DCAT-AP HVD spec, both options seem to be acceptable.
Regarding contact points, I just read the PR, I suppose that if a dataset has a contact point (or its user / organisation has a contact point) then 
all the API / Dataservices which are part of a dataset, will inherit this contact point?
I am just wondering about the cardinality, which is 1..* for a contact point on a Dataservice. Will this field be somehow  mandatory on the creation of an API?
The approach 1 on the DataServices seems the most flexible, as it will allow to manage the required extra metadata.

Best regards

**Reply by maudetes** [reply]
Thank you for your feedback!
Indeed, we may deduce the contact point from the dataset if not specified otherwise, but we must be able to have a specific contact point value for the API that may differ from the dataset.
Regarding the cardinality of the contact point, we will only support one contact point at first but we haven't yet discussed make it mandatory. Since HVD producers have to follow some metadata guidelines in any case, we could rely on these guidelines if we don't want to make it mandatory by default for all.

**Reply by AlainVagner** [reply]
Looks perfect to me! Thanks a lot for all your efforts.
